tests/test_minibook.py

```
@@ -47,6 +47,9 @@ def test_generate_html(tmp_path):
     # Check that the title, description, and links are in the content
     assert title in content
     assert description in content
+    print(content)
+    assert False
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)